tests/test_tasks.py

```diff
@@ -103,3 +103,81 @@ def test_NoToken(self):
     def test_InvalidToken(self):
         response = self.get(self.VALID_URL, self.bogus_token())
         self.assert_invalid_token(response)
+
+
+class TestTaskPost(EndpointTestCase):
+    """
+    Tests for '/api/students/<int:student_id>/tasks/' [POST] endpoint.
+    """
+
+    # Fixture specific URL to available student data.
+    VALID_URL = "/api/students/2/tasks/"
+    # Fixture specific URL to student data not available for current user.
+    NOT_PERMITTED_URL = "/api/students/1/tasks/"
+    # Fixture specific URL to invalid student ID.
+    STUDENT_NOT_FOUND_URL = "/api/students/12345/tasks/"
+    DATA_JSON_POST = {
+        "name": "Another new task to do",
+        "value": 21,
+    }
+
+    def test_Success(self):
+        # Send POST data and response HTTPResponse:
+        response = self.post(
+            self.VALID_URL, data=self.DATA_JSON_POST, token=self.access_token()
```

Comment:
`self.post` use `self.access_token` by default, remove `token=...`.

Reply:
Ok

Reply:
Ok I see -> access_token = token if token is not None else self.access_token()

Reply:
In get We have the same defaults with token . Only digression and subjective opinion - look better default params with some data classes or pydantic.